content/en/docs/ops/deployment/multicluster-considerations/index.md

```diff
@@ -0,0 +1,161 @@
+without exposing the control plane on the public internet. Such an option, if it exists and meets your requirements,
+will likely be the best choice.
+
+{{< image width="80%"
+    link="remote-clusters.svg"
+    caption="Remote clusters"
+    >}}
+
+## DNS and Service Deployment
+
+When a client application makes a request to some host, it must first perform a DNS lookup for the hostname to
+obtain an IP address before it can proceed with the request.
+
+In Kubernetes, the DNS server residing within the cluster typically handles this DNS lookup. Kubernetes
+`Service` configures the DNS server.
+
+Istio ignores the IP address returned from the DNS lookup, however. Instead, it
+uses the list of active endpoints for the host and load balances across them. Istio uses either Kubernetes
+`Service`/`Endpoint` or Istio `ServiceEntry` to configure its internal mapping of hostname to IP addresses.
+
+This two-tiered naming system becomes more complicated when you have multiple clusters. Istio is inherently
+multicluster-aware, but Kubernetes is not (today). Because of this, the client cluster must have a DNS entry
+for the service in order for the DNS lookup to succeed, and a request to be successfully sent. This is true
+even if there are no instances of that service's Pods running in the client cluster.
+
+To ensure that DNS lookup succeeds, you must deploy a Kubernetes `Service` to each cluster that consumes that
+service. This ensures that regardless of where the request originates, it will pass DNS lookup and be handed
+to Istio for proper routing.
+
+This can also be achieved with Istio `ServiceEntry`, rather than Kubernetes `Service`. `ServiceEntry` does not
+configure DNS, however. This means that DNS will have to be configured either manually or with automated
+tooling, such as the [Istio CoreDNS Plugin](https://github.com/istio-ecosystem/istio-coredns-plugin).
+
+{{< tip >}}
+    There are a few efforts in progress that will help simplify the DNS story in the future:
+
+    * Istio will soon support DNS interception for workloads with a sidecar proxy. This will allow Istio to perform
+    DNS lookup on behalf of the application.
+    * [Admiral](https://github.com/istio-ecosystem/admiral) is an Istio community project that provides a number of
+    multicluster capabilities, including automatic creation of service DNS entries.
+    * [Kubernetes Multi-Cluster Services](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api)
+    is a Kubernetes Enhancement Proposal (KEP) that defines an API for exporting services to multiple clusters.
+{{< /tip >}}
+
+## Networks
+
+Istio uses a simplified definition of [network](/docs/ops/deployment/deployment-models#network-models) to refer to
+pods that have direct reachability. For example, by default all the pods in a single cluster are on a single network.
```

Comment:
This sounds strange. Where's the `simplified definition of network`? The wording makes it expected at the link target. Maybe there should be a glossary entry with this simplified definition? Or maybe just say it here:
```suggestion
All pods in an Istio [network](/docs/ops/deployment/deployment-models#network-models) can reach each other directly. For example, by default all the pods in a single cluster are on a single network.
```

Reply:
@frankbu I'm not too sure about the revised wording. It makes it sound that Istio implements a network, rather than just defining a term/concept.

Reply:
I suppose that's true. Maybe we really need to add a glossary entry for network with the "Istio definition". Then, we could use your wording, but using the glossary reference, instead of a link to the other doc

Reply:
I've added a glossary entry in this PR.